apps/examples/st_things/Makefile

```diff
@@ -28,17 +28,17 @@ THREADEXEC = TASH_EXECMD_SYNC
 
 # SmartThings Things example
 
+ROOTDEPPATH = --dep-path .
 ASRCS =
 CSRCS =
-ifeq ($(CONFIG_EXAMPLES_ST_THINGS_LIGHT),y)
-CSRCS += light/sample_light_things.c light/sample_light_user.c
-endif
-ifeq ($(CONFIG_EXAMPLES_ST_THINGS_BLINK),y)
-CSRCS += blink/sample_blink_things.c blink/sample_blink_user.c blink/iotbus_util.c
-endif
-ifeq ($(CONFIG_EXAMPLES_ST_THINGS_MSG_HANDLING),y)
-CSRCS += msghandling/sample_msghandling_things.c msghandling/sample_msghandling_user.c
-endif
+
+BUILDIRS   := $(dir $(wildcard */Make.defs))
+
+define Add_Framework
```

Comment:
Could you rename the definition? This is not a framework.

Reply:
done.